src/plugins/lightbox/lightbox.js

```diff
@@ -142,10 +143,42 @@ var componentName = "wb-lbx",
 					var $item = this.currItem,
 						$content = this.contentContainer,
 						$wrap = this.wrap,
+						$modal = $wrap.find( ".modal-dialog" ),
 						$buttons = $wrap.find( ".mfp-close, .mfp-arrow" ),
 						len = $buttons.length,
 						i, button;
 
+					if ( $modal && $modal.length !== 0 ) {
+						var footer = $modal.find( ".modal-footer" )[ 0 ],
+							hasFooter = ( footer && footer.length !== 0 ) ? true : false,
```

Comment:
With `( footer && footer.length !== 0 )` maybe it should be `( footer !== null )`.